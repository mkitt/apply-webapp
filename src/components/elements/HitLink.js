// @flow
import glamorous from 'glamorous'
import Link from './Link'

export default glamorous(Link)({
  position: 'absolute',
  top: 0,
  right: 0,
  bottom: 0,
  left: 0,
  zIndex: 1,
  backgroundColor: 'rgba(0, 0, 0, 0)',
})
