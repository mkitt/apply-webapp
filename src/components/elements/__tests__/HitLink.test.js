import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import HitLink from '../HitLink'

describe('<HitLink />', () => {
  it('shows the correct snapshot tree for the <HitLink> element', () => {
    const component = (
      <MemoryRouter>
        <HitLink to="/hello">The HitLink</HitLink>
      </MemoryRouter>
    )
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
