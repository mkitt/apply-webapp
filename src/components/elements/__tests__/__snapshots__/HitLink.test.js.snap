// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HitLink /> shows the correct snapshot tree for the <HitLink> element 1`] = `
.css-0,
[data-css-0] {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 1;
  background-color: rgba(0, 0, 0, 0);
}

<a
  className="css-0"
  href="/hello"
  onClick={[Function]}
>
  The HitLink
</a>
`;
